pro/src/components/CollectiveOfferSummary/CollectiveOfferSummary.tsx

```diff
@@ -33,13 +34,14 @@ const CollectiveOfferSummary = ({
   stockEditLink,
   visibilityEditLink,
 }: ICollectiveOfferSummaryProps) => {
+  const offerPublic = isCollectiveOffer(offer) && !offer.isPublicApi
   return (
     <>
       <SummaryLayout>
         <SummaryLayout.Content fullWidth>
           <SummaryLayout.Section
             title="Détails de l’offre"
-            editLink={offerEditLink}
+            editLink={offerPublic && offerEditLink}
```

Comment:
Le champ est optionel. Un truc comme ça serait peut être mieux, je pense.
```suggestion
            editLink={offerPublic ? offerEditLink : null}
```

Reply:
merci c'est fait